WordPress/src/main/java/org/wordpress/android/ui/SlidingScrollView.java

```diff
@@ -0,0 +1,51 @@
+package org.wordpress.android.ui;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.ViewTreeObserver;
+import android.widget.ScrollView;
+
+public class SlidingScrollView extends ScrollView {
```

Comment:
Might be worth pointing to `SlidingRelativeLayout` as a comment as they're using the same implementation.